test/scrabble_test.rb

```diff
@@ -4,9 +4,22 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
```

Comment:
I would have expected at least 5 tests in this file:

test_it_can_score_a_single_letter
test_it_can_score_a_word
test_it_can_score_a_word_with_letter_multipliers
test_it_can_score_a_word_with_word_multiplier
test_it_can_score_a_word_greater_than_7_letters